src/components/ItaliaTheme/View/Commons/SideMenu.jsx

```diff
@@ -151,7 +155,7 @@ const SideMenu = ({ data, content_uid }) => {
 
   return headers?.length > 0 ? (
     <div className="sticky-wrapper navbar-wrapper page-side-menu">
-      <nav className="navbar it-navscroll-wrapper navbar-expand-lg">
+      <div className="navbar it-navscroll-wrapper navbar-expand-lg">
```

Comment:
non sono sicura sia la mossa giusta cambiare il nav qui, vedi commento lasciato in BandoView